Create a second button in the page, support external coordinates json

ceres-c opened the pull request:
Hey, while using your script I noticed that sometimes it did not work when the `kmtBoot` object did not have embedded properties but rather a link to a json with then coordinates, so I have added code to handle that
Also, I have modified the script to create another button besides the "Save route", rather than modifying the save button itself

DreiDe commented:
Hi @ceres-c, thanks for the PR. It was open for a long time as I basically abandoned the project. But as there seems to be interest in the plugin and even PRs like yours it would be nice to have it working again. Would you be willing to make this PR work with the current base branch? I think a seperate Button makes sense.

ceres-c committed:
Support tours+collections+planner, download gpx button, fetch filename, more download methods

ceres-c commented:
Done, updated the script and revamped it a bit. I have added support for:
1) Tour pages (no need to go to the planner page)
2) Collections pages (download each part of the collection)
3) Filenames are generated from the tour/collection part names. Does not apply to the planner as there is no defined name there

I've also updated the readme with more screenshots

ceres-c committed:
Update readme with screenshots

DreiDe commented:
Very cool @ceres-c I will merge it. Thanks for implementing!

ceres-c commented:
Great :)
I haven't tested it with firefox, but it might probably work just as well there. Maybe there might be a bit of wrestling to do with manifests, or at least that was the case when I made my last browser extension years ago.
Would you be interested in publishing this on chrome/firefox stores? It might help a lot of people, I think. I personally know people who would have loved the convenience after I told them about this extension.

DreiDe commented:
@ceres-c I never officially published an extension, but it shouldn't be that hard I guess. I will have a look :)
